preference-service/src/routes/preference.route.ts

```diff
@@ -0,0 +1,16 @@
+import { Router } from "express";
+import { PreferenceController } from "../controllers/preference.controller";
+
+
+function PreferenceRouter(): Router {
+    const router = Router();
+
+    router.post("/create", PreferenceController.createPreference);
+    router.get("/", PreferenceController.getAllPreferences);
+    router.get("/:id", PreferenceController.getPreferenceById);
+    router.get("/user/:userId", PreferenceController.getPreferenceByUserId);
+    router.get("/place-categories/:placeCategoryId", PreferenceController.getPreferenceByPlaceCategoryId);
```

Comment:
Following the pattern, I believe it would be place-category

Reply:
All endpoints in core-api are in plural form... I suppose it's better to change `/user/` to `/users/`

Reply:
Done

Reply:
Done